src/components/BorrowerProfile/UpdateDetails.vue

```diff
@@ -57,9 +69,24 @@ export default {
 			default: '',
 		},
 	},
+	data() {
+		return {
+			readMoreClicked: false
+		};
+	},
 	computed: {
 		formattedJournalDate() {
 			return format(parseISO(this.date), 'MMMM dd, yyyy');
+		},
+		truncateBody() {
+			const bodyLength = this.htmlSafeBody?.length ?? 0;
+			return bodyLength > 210;
+		},
+		shortBody() {
+			return `${this.htmlSafeBody?.substring(0, 210)}... `;
+		},
+		htmlSafeBody() {
+			return DOMPurify.sanitize(this.body, { ALLOWED_TAGS: ['b', 'i', 'em', 'strong', 'a', 'p', 'br'] });
```

Comment:
Would there be H_n_ header elements in here?

Reply:
Hmm yeah, there could possibly be. I'm not sure where this html is coming from..... I'll add h tags as allowed here as well.